orc8r/cloud/go/services/configurator/storage/storage.proto

```diff
@@ -208,11 +208,20 @@ message EntityLoadCriteria {
     bool load_assocs_from_this = 4;
 
     bool load_permissions = 5;
+
+    // The following parameters allow for pagination of entity loads.
+    // Size of page to return
```

Comment:
Throughout this proto file: follow go-style doc conventions in proto files. So e.g. here "page_size is ..."